extensions/lua/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "zed_lua"
-version = "0.0.3"
+version = "0.0.4"
```

```suggestion
version = "0.0.3"
```

Versions are bumped in separate PRs, see https://github.com/zed-industries/zed/pull/17200#discussion_r1739743817 for example.